README.md

```diff
@@ -1,30 +1,43 @@
-# MeiliSearch
+<p align="center">
+  <img src="assets/logo.png" alt="MeiliSearch" />
+</p>
+
+<h1 align="center">MeiliSearch</h1>
+
+<h4 align="center">
+  <a href="https://www.meilisearch.com">Website</a> | 
+  <a href="https://blog.meilisearch.com">Blog</a> | 
+  <a href="https://fr.linkedin.com/company/meilisearch">LinkedIn</a> | 
+  <a href="https://twitter.com/meilisearch">Twitter</a> | 
+  <a href="https://docs.meilisearch.com">Documentation</a> | 
+  <a href="https://docs.meilisearch.com/resources/faq.html">FAQ</a>
+</h4>
 
 [![Build Status](https://github.com/meilisearch/MeiliSearch/workflows/Cargo%20test/badge.svg)](https://github.com/meilisearch/MeiliSearch/actions)
 [![dependency status](https://deps.rs/repo/github/meilisearch/MeiliSearch/status.svg)](https://deps.rs/repo/github/meilisearch/MeiliSearch)
 [![License](https://img.shields.io/badge/license-MIT-informational)](https://github.com/meilisearch/MeiliSearch/blob/master/LICENSE)
 [![Slack](https://img.shields.io/badge/slack-MeiliSearch-blue.svg?logo=slack)](https://slack.meilisearch.com)
```

Comment:
Hey @matboivin, 
As you said early, could you align the badges to follow the other links please ? :)

Reply:
Updated! :D